### ISSUE TITLE
Seven Theme - Dashboard

### ISSUE THREAD

**Issue by BossOfGames** [BossOfGames/phpvms#15]

**Comment by arthurpar06** [comment]
Looks great! I just have one question about the weather widget: do we want to keep the padding around the table, or should it be a full-width table?

Current:
![Image](https://github.com/user-attachments/assets/c59b467b-5568-4fd6-b429-bfccefac438b)

Full-width table (just need to fix the border bottom):
![Image](https://github.com/user-attachments/assets/3db5f77c-4bd1-4cdd-ae00-64c5d780d5c5)

**Comment by BossOfGames** [comment]
@arthurpar06 better question is do we want to keep the table 🤣 

There might be a better, more concise way, of presenting that widget (it uses up a lot of space, maybe simply showing the raw metar/taf, then a parsed one in a separate tab). Maybe we make that an enhancement?

**Comment by BossOfGames** [comment]
If we want to keep the table, I agree with your implementation :)

**Comment by arthurpar06** [comment]
Yeah, we could have two tabs: a raw tab and a decoded tab. Then, in those tabs, we can have tables (maybe full-width?)

**Comment by BossOfGames** [comment]
> Yeah, we could have two tabs: a raw tab and a decoded tab. Then, in those tabs, we can have tables (maybe full-width?)

Yes, the table full width would be fine. Though, I would almost recommend making a flex container and adding a little design to it. I have some ideas and some old code sitting on the shelf with a descent design I might whip out.

see https://github.com/BossOfGames/SimVector/blob/master/resources/js/components/AirportWXTab.vue

Edit: The live version for the above code doesn't work because wx api keys weren't updated on the live site, plus it's being sunset.

**Comment by arthurpar06** [comment]
okay then i'll let you handle it

**Comment by BossOfGames** [comment]
> okay then i'll let you handle it

Actually, I just had a thought. Could we just embed a https://metar-taf.com/ feed? Or does that carry some additional baggage I'm not aware of?

**Comment by arthurpar06** [comment]
We can, I guess it's just adding their JS/CSS code

**Comment by ArthurHetem** [comment]
![Image](https://github.com/user-attachments/assets/6e0e9ec2-831c-40e3-b0e6-71469c5bf5f4)
![Image](https://github.com/user-attachments/assets/0d75bf63-df89-4f52-b670-7695db840cae)

What about something like this?

**Comment by BossOfGames** [comment]
Oh, that could work! Though, have you checked if the embed from metar-taf works on dark mode?

**Comment by ArthurHetem** [comment]
As we talked, i made the changes to the Weather Widget, making it a button with bs5 Offcanvas component:
![Image](https://github.com/user-attachments/assets/afc66912-8542-4369-9297-87aed5d9c434)

**Comment by BossOfGames** [comment]
Looks great!

One more thing of note. I know that we have the blue headers here. Should we apply blue headers to everything around the theme? Or should we remove the blue headers to keep it in-line with the other sections of code?

Doing the latter will help keep this a "reference theme"

**Comment by ArthurHetem** [comment]
We've changed the headers to all blue, closing this
